shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/RuleServiceImpl.java

```diff
@@ -302,7 +302,17 @@ public Boolean enabledByIdsAndNamespaceId(final List<String> ids, final Boolean
             RuleDO before = JsonUtils.jsonToObject(JsonUtils.toJson(ruleDO), RuleDO.class);
             ruleDO.setEnabled(enabled);
             if (ruleMapper.updateEnable(id, enabled) > 0) {
-                ruleEventPublisher.onUpdated(ruleDO, before);
+                List<RuleConditionDO> conditionList = ruleConditionMapper.selectByQuery(new RuleConditionQuery(ruleDO.getId()));
```

Comment:
this method just modify the status , the conditions not changed, why doing this

Reply:
Condition is a part of the selector and rule objects, and currently only supports pushing a complete object to Boostrap, unless a new push event type identifier is added to enable switching

Reply:
Yes, if don't put condition into the SelectorData object and send the event

Reply:
hi, i found when change status of selectors or rules, data changed will not be sent to bootstrap. will u change this too?

Reply:
hi, i found when changing status of selector or rules, no data change event will be sent to bootstrap, which may be a mistake. could u pls check this again, and fix it

Reply:
This PR is to fix the issue when the selector and rule are not push data change event to bootstrap when modifying the enable status. I don't know what mean about you say the changing status, did I miss some situation?

Reply:
by 'changing stauts' i mean change selector enabled status